constants.py

```diff
@@ -685,3 +685,31 @@
 
 kVelocityControlMotorType = DCMotor.falcon500()
 kVelocityControlkV = 0.01
+
+# Elevator constants, replace values
+
+kElevator1CANID = 60
+kElevator1Name = "Elevator1Motor"
+kElevator1PGain = 0.12
+kElevator1IGain = 0
+kElevator1DGain = 0
+kElevator1RotationAmount = 3000
+kElevator1Inverted = False
+
+kElevator2CANID = 60
+kElevator2Name = "Elevator2Motor"
+kElevator2PGain = 0.12
+kElevator2IGain = 0
+kElevator2DGain = 0
+kElevator2RotationAmount = 3000
+kElevator2Inverted = True
+
+kMotorPulleyGearRatio = 60 / 18
+
+kPulleyGearPitchDiameter = 1.504
+"""inches"""
+
+kBottomPositionBeltPosition = 0
+kAmpPositionBeltPosition = 0.5
+kTopPositionBeltPosition = 1
```

Comment:
from cad, top position is 26.5 inches, amp position is 19.125 inches
amp position could change but use these values